concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-134	134	there is no X such that X...	there is no X such that X...			don't know what to do with this one
+Zhang-2019-308-135	135	the thighbone	thighbone	800	THIGH	
+Zhang-2019-308-136	136	be thin	thin	2249	THIN (OF SHAPE OF OBJECT)	
+Zhang-2019-308-137	137	thou	thou	1215	THOU	
+Zhang-2019-308-138	138	three	three	492	THREE	
+Zhang-2019-308-139	139	to beat	to beat	2133	STRIKE OR BEAT	
+Zhang-2019-308-140	140	to bend	to bend	2224	BEND (SOMETHING)	
+Zhang-2019-308-141	141	to break (vi)	to break (vi)	680	BREAK (DESTROY OR GET DESTROYED)	
+Zhang-2019-308-142	142	to break (vt)	to break (vt)	680	BREAK (DESTROY OR GET DESTROYED)	
+Zhang-2019-308-143	143	to carry water on back	to carry water on back			too specific to map
+Zhang-2019-308-144	144	to chase	to chase	1047	PURSUE	
+Zhang-2019-308-145	145	to chew	to chew	321	CHEW	
+Zhang-2019-308-146	146	to confer	to confer			
+Zhang-2019-308-147	147	to curve	to curve	2224	BEND (SOMETHING)	
+Zhang-2019-308-148	148	to defeat	to defeat	782	DEFEAT	
+Zhang-2019-308-149	149	to descend	to descend	2014	GO DOWN (DESCEND)	
+Zhang-2019-308-150	150	to destroy	to destroy	681	DESTROY	
+Zhang-2019-308-151	151	to die	to die	1494	DIE	
+Zhang-2019-308-152	152	to dig out (earth)	to dig out (earth)	1418	DIG	
+Zhang-2019-308-153	153	to drop, drop	to drop, drop	808	DRIP (FALL IN GLOBULES)	
+Zhang-2019-308-154	154	to extinguish	to extinguish	127	EXTINGUISH	
+Zhang-2019-308-155	155	to fall	to fall	1280	FALL	
+Zhang-2019-308-156	156	to fall (from hand)	to fall (from hand)			
+Zhang-2019-308-157	157	to fry, roast	to fry, roast	2146	ROAST OR FRY	
+Zhang-2019-308-158	158	to give	to give	1447	GIVE	
+Zhang-2019-308-159	159	to hit	to hit	1433	HIT	
+Zhang-2019-308-160	160	to itch	to itch	148	ITCH	
+Zhang-2019-308-161	161	to kill	to kill	1417	KILL	
+Zhang-2019-308-162	162	to lose	to lose	1660	LOSE	
+Zhang-2019-308-163	163	to oppose, disturb	to oppose, disturb	2883	DISTURB	
+Zhang-2019-308-164	164	to peck	to peck	3684	PECK	
+Zhang-2019-308-165	165	to plant	to plant	1486	PLANT (SOMETHING)	
+Zhang-2019-308-166	166	to press	to press	1913	PRESS	
+Zhang-2019-308-167	167	to revive	to revive			
+Zhang-2019-308-168	168	to scoop water	to scoop water			
+Zhang-2019-308-169	169	to scrape	to scrape	1767	SCRAPE	
+Zhang-2019-308-170	170	to scratch	to scratch	1436	SCRATCH	
+Zhang-2019-308-171	171	to shout	to shout	715	SHOUT	
+Zhang-2019-308-172	172	to sit	to sit	1416	SIT	
+Zhang-2019-308-173	173	to slip	to slip	1324	SLIP	
+Zhang-2019-308-174	174	to smoke	to smoke	1689	SMOKE (INHALE)	
+Zhang-2019-308-175	175	to stand	to stand	1442	STAND	
+Zhang-2019-308-176	176	to take care of, protect	to take care of, protect			
+Zhang-2019-308-177	177	to take off	to take off	3538	TAKE OFF	
+Zhang-2019-308-178	178	to think	to think	2271	THINK	
+Zhang-2019-308-179	179	to wear	to wear	976	WEAR	
+Zhang-2019-308-180	180	to weave	to weave	133	WEAVE	
+Zhang-2019-308-181	181	the tree	tree	906	TREE	
+Zhang-2019-308-182	182	to twist, to spin	twist, spin	430	TWIST	
```

Comment:
TWIST is "To wind something around something else or several things around each other." - does this describe what's meant here?

Reply:
I'd say yes.